modules/command-engine/core/src/test/scala/surge/core/SurgeCommandServiceSinkSpec.scala

```diff
@@ -4,15 +4,23 @@ package surge.core
 
 import akka.actor.{ ActorSystem, NoSerializationVerificationNeeded }
 import akka.testkit.{ TestKit, TestProbe }
+import org.scalatest.BeforeAndAfterAll
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpecLike
 import surge.core.command.{ SurgeCommandServiceSink, SurgeMultiCommandServiceSink }
 import surge.streams.sink.TestEventSource
 
 import java.util.UUID
+import scala.concurrent.duration.DurationInt
 import scala.concurrent.{ ExecutionContext, Future }
 
-class SurgeCommandServiceSinkSpec extends TestKit(ActorSystem("SurgeCommandServiceSinkSpec")) with AnyWordSpecLike with Matchers {
+class SurgeCommandServiceSinkSpec extends TestKit(ActorSystem("SurgeCommandServiceSinkSpec")) with AnyWordSpecLike with Matchers with BeforeAndAfterAll {
+
+  override def afterAll(): Unit = {
+    TestKit.shutdownActorSystem(system, verifySystemShutdown = true)
+    super.afterAll()
```

Comment:
I think this `super.afterAll()` can be removed since the default implementation in the superclass is to do nothing

Reply:
you are right

Reply:
done, removed